### ISSUE TITLE
Titan One issues

### ISSUE THREAD

**Issue by oneswitch** [evilC/UCR#131]
Sorry, Clive! A couple more issues...

1. Noticing some odd vibration being passed through to an attached PS4 controller (via the INPUT socket) when using UCR.

2. The attached PS4 joypad is not usable when controlled via the UCR/Titan One method. Would be very handy if this could be. It is in GTuner MaxAim D.I. normally.

**Comment by oneswitch** [comment]
4. If you start UCR without a Titan One attached, it's very slow to start. Not a big problem, but might be nice to resolve in some way.

**Comment by oneswitch** [comment]
And to add to the Titan One pot, I wondered if this link might be useful, regarding converting a stick to six-axis motion control:

[http://www.consoletuner.com/forum/viewtopic.php?f=23&t=4767&p=36686#p36684](url)

**Comment by evilC** [comment]
My plan is to add an option to enable / disable T1, and it defaults to off.
Added as issue #134

BTW the Link is dead - could you create a new issue for the other item and post a working link in it?
Once that is done I can close this ticket as we can use #134 to track.
